### PR TITLE
fixed issue #1299 on the app

### LINKED ISSUE #1299: Add update alert icon to calendar events that have recently been updated
this was existing functionality in 2.4; for any event with changes logged in the prior _n_ days (_n_ being configurable, default = 5), the alert icon will appear in the calendar (upper corner for month/day/week views). We should add it as well to the event view (on the items/categories updated), as well as to the event list view.

icon:  `fa-exclamation` or `fa-exclamation-circle`; color=red

Items which trigger the alert:

- change to start time
- change to end time
- change to location
- change to instructor
- change to objectives 
- change to learning materials 
- change to description

### PR DESCRIPTION
**Pull request by djvoa12**
This is a partial fix for #1299. The rest are in the calendar addon PR.
- Shows icon on `dashboard-agenda`.

screen shot 2016-01-27 at 3 53 53 pm

Fixes #1299

### CODE REVIEW

**Comment by jrjohnson** [comment]
@djvoa12 looks like a JSHint issue.

**Comment by jrjohnson** [comment]
@saschaben please test.

**Comment by saschaben** [comment]
seems to be OK. Unable to validate that alerts are showing for the correct events though since all our demo events are showing as updated for mr. gutierrez.

any chance we can get a mouseover tooltip type value stating "updated"? if super easy (as in `alt="updated"`) then let's get it in. if not, let's add a new enhancement ticket.

File: app/styles/components/_recently-updated-display.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.recently-updated-display {
```
Comment (jrjohnson) [inline app/styles/components/_recently-updated-display.scss]: @djvoa12 this is duplicate style from ilios-calendar - we shouldn't need it in both places.

**Comment by jrjohnson** [comment]
Contained in #1369

**Pull request closed without merging**